src/cascadia/TerminalSettingsEditor/ColorSchemeViewModel.idl

```diff
@@ -5,11 +5,14 @@ import "ColorSchemes.idl";
 
 namespace Microsoft.Terminal.Settings.Editor
 {
-    runtimeclass ColorSchemeViewModel : Windows.UI.Xaml.Data.INotifyPropertyChanged
+    runtimeclass ColorSchemeViewModel : Windows.UI.Xaml.Data.INotifyPropertyChanged, Windows.Foundation.IStringable
     {
-        ColorSchemeViewModel(Microsoft.Terminal.Settings.Model.ColorScheme scheme);
+        ColorSchemeViewModel(Microsoft.Terminal.Settings.Model.ColorScheme scheme, ColorSchemesPageViewModel parentPageVM);

```

Comment:
This doesn't need to be part of the public interface! Actually, the whole constructor can be removed from the IDL. Only put things in an IDL if somebody should be allowed to construct them from a different DLL!